src/frequenz/client/dispatch/test/fixtures.py

```diff
@@ -0,0 +1,28 @@
+# License: MIT
+# Copyright © 2024 Frequenz Energy-as-a-Service GmbH
+
+"""Useful fixtures for testing."""
+
+from pytest import fixture
+
+from ..types import Dispatch
+from .client import FakeClient
+from .generator import DispatchGenerator
+
+
+@fixture
+def sample() -> Dispatch:
+    """Return a dispatch sample."""
+    return DispatchGenerator().generate_dispatch()
+
+
+@fixture
+def generator() -> DispatchGenerator:
+    """Return a dispatch generator."""
+    return DispatchGenerator()
```

Comment:
No need to create 2 generators, no?

```suggestion
@fixture
def generator() -> DispatchGenerator:
    """Return a dispatch generator."""
    return DispatchGenerator()


@fixture
def sample(generator: DispatchGenerator) -> Dispatch:
    """Return a dispatch sample."""
    return generator.generate_dispatch()
```

This way if you use both fixtures the generator will be reused.